exp/internal/controllers/machinepool_controller_phases_test.go

```diff
@@ -1335,10 +1336,8 @@ func TestReconcileMachinePoolMachines(t *testing.T) {
 					g.Expect(machineList.Items).To(HaveLen(len(tc.infraMachines)))
 					for i := range machineList.Items {
 						machine := &machineList.Items[i]
-						infraMachine, err := external.Get(ctx, r.Client, &machine.Spec.InfrastructureRef, machine.Namespace)
+						_, err := external.Get(ctx, r.Client, &machine.Spec.InfrastructureRef, machine.Namespace)
 						g.Expect(err).ToNot(HaveOccurred())
-
-						g.Expect(util.IsControlledBy(infraMachine, machine)).To(BeTrue())
```

Comment:
why was this test case removed?

Reply:
Since we removed the code in the MachinePool controller to set the ownerRef of the infraMachine, the infraMachine would not have an ownerRef to the Machine by running the MachinePool controller alone. The Machine controller, which is already doing that, is not being run in this test so it would not take effect here.